code/datums/mutations/body.dm

```diff
@@ -147,3 +160,57 @@
 /datum/mutation/human/race/on_losing(mob/living/carbon/monkey/owner)
 	if(owner && istype(owner) && owner.stat != DEAD && (owner.dna.mutations.Remove(src)))
 		. = owner.humanize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+
+/datum/mutation/human/glow
+	name = "Glowy"
+	desc = "You permanently emit a light with a random color and intensity."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>Your skin begins to glow softly.</span>"
+	instability = 5
+	var/obj/effect/dummy/luminescent_glow/glowth //shamelessly copied from luminescents
+	var/glow = 1.5
+
+/datum/mutation/human/glow/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	glowth = new(owner)
+	glowth.set_light(glow, glow, dna.features["mcolor"])
+
+/datum/mutation/human/glow/on_losing(mob/living/carbon/human/owner)
+	if(..())
+		return
+	qdel(glowth)
+
+/datum/mutation/human/strong
+	name = "Strength"
+	desc = "The user's muscles slightly expand."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>You feel strong.</span>"
+	difficulty = 16
```

Comment:
does this do anything?

Reply:
Need to check but iirc on goon the effects for it are all on objects

Reply:
oh wait, read OP

Reply:
oh i see, it's just for crafting hulk